src/shared/database/setup.ts

```diff
@@ -205,6 +206,35 @@ async function setupNetworksTable(): Promise<void> {
   }
 }
 
+async function setupAmendmentsEnabledTable(): Promise<void> {
+  const hasAmendmentsEnabled = await db().schema.hasTable('amendments_enabled')
+  if (!hasAmendmentsEnabled) {
+    await db().schema.createTable('amendments_enabled', (table) => {
+      table.string('amendment_id')
+      table.string('networks')
+      table.integer('ledger_index')
+      table.string('tx_hash')
+      table.dateTime('date')
+      table.primary(['amendment_id', 'networks'])
+    })
+  }
+  if (!(await db().schema.hasColumn('amendments_enabled', 'tx_hash'))) {
+    await db().schema.alterTable('amendments_enabled', (table) => {
+      table.string('tx_hash').after('networks')
+    })
+  }
+  if (!(await db().schema.hasColumn('amendments_enabled', 'date'))) {
+    await db().schema.alterTable('amendments_enabled', (table) => {
+      table.datetime('date').after('tx_hash')
+    })
+  }
+  if (!(await db().schema.hasColumn('amendments_enabled', 'ledger_index'))) {
+    await db().schema.alterTable('amendments_enabled', (table) => {
+      table.integer('ledger_index').after('networks')
+    })
+  }
```

Comment:
Was this just needed for testing? It won't be needed in prod, since the db doesn't exist yet.